src/bin/cargo/cli.rs

```diff
@@ -32,7 +32,7 @@ Available unstable (nightly-only) flags:
     -Z avoid-dev-deps   -- Avoid installing dev-dependencies if possible
     -Z minimal-versions -- Install minimal dependency versions instead of maximum
     -Z no-index-update  -- Do not update the registry, avoids a network request for benchmarking
-    -Z unstable-options -- Allow the usage of unstable options such as --registry
+    -Z unstable-options -- Allow the usage of unstable options such as --build-plan
```

Comment:
Hm, maybe build-plan isn't such a good alternative given #7614.  Maybe just remove the "such as…", as this seems like something that will be difficult to keep up-to-date as options get stabilized.

Reply:
> Maybe just remove the "such as…", as this seems like something that will be difficult to keep up-to-date as options get stabilized.

I agree with you. It seems best to remove "such as ...".